docs/mine-hnt/validators/penalties.mdx

```diff
@@ -79,41 +81,44 @@ Learn more about [Distributed Key Generation](https://en.wikipedia.org/wiki/Dist
+
+- This penalty can occur if a majority of other Validators in the consensus group report your Validator as not being "seen"
+- This is equivalent to the Validator being offline, not available, or not communicating with the group.
+- This penalty is applied for each occurrence, which can happen multiple times during a consensus group.
+- The amount applied per occurrence is a chain variable, currently set at `0.33`. The variable is viewable by running `miner ledger variables election_seen_penalty`.
 
 **BBA (Byzantine Binary Agreement) Penalties**
-* This penalty can occur when the Validator is seen but fails to contribute a bundle of transactions and metadata to the block.  
-* Acquiring this penalty usually means the Validator is slow/underpowered, or poorly connected.
-* This may also occur if the time on your Validator is skewed vs the group time.  [Run `ntp`](https://en.wikipedia.org/wiki/Network_Time_Protocol)!
-* This penalty is applied for each occurrence, which can happen multiple times during a consensus group.
-* The amount applied per occurrence is a chain variable, currently set at `0.1`. The variable is viewable by running `miner ledger variables election_bba_penalty`.
+
+- This penalty can occur when the Validator is seen but fails to contribute a bundle of transactions and metadata to the block.
+- Acquiring this penalty usually means the Validator is slow/underpowered, or poorly connected.
+- This may also occur if the time on your Validator is skewed vs the group time. [Run `ntp`](https://en.wikipedia.org/wiki/Network_Time_Protocol)!
+- This penalty is applied for each occurrence, which can happen multiple times during a consensus group.
+- The amount applied per occurrence is a chain variable, currently set at `0.1`. The variable is viewable by running `miner ledger variables election_bba_penalty`.
 
 :::info
 
-Learn more about [Byzantine Agreements](https://en.wikipedia.org/wiki/Byzantine_fault) and their affect on [Consensus](https://en.wikipedia.org/wiki/Consensus_(computer_science))
+Learn more about [Byzantine Agreements](https://en.wikipedia.org/wiki/Byzantine_fault) and their affect on [Consensus](<https://en.wikipedia.org/wiki/Consensus_(computer_science)>)
 
 :::
 
 ## Penalty Decay
 
-Penalties decay over time.  
+Penalties decay as blocks are formed.
 
-Each penalty, when assessed, also decays at the same rate.  This decay rate is in blocks, and is stored as a chain variable. View the current decay block limit by running `miner ledger variables penalty_history_limit`.  
+Each penalty, when assessed, also decays at the same rate. This decay rate is in blocks, and is stored as a chain variable. View the current decay block limit by running `miner ledger variables penalty_history_limit`. This value is set to `10080` blocks for mainnet, or a target of 1 week.
```

Comment:
```suggestion
Each penalty accrued decays to 0 in approximately 1 week regardless of its initial magnitude. This decay rate is in blocks, and is stored as a chain variable. View the current decay block limit by running `miner ledger variables penalty_history_limit`. This value is set to `10080` blocks for mainnet.
```

Reply:
I'll make some text changes here, but don't want to show a time.  It's a block based count and there is no certainty that blocks will average to 60s over that time period.

Reply:
Thanks @jdgemm, updates made.